.github/workflows/rust-wasmedge-sdk-docs.yml

```diff
@@ -7,7 +7,7 @@ concurrency:
 on:
   push:
     branches:
-      - "rust/*"
+      - "release/rust/*"
```

Comment:
The release tag of sdk should be `rust-sdk/*`.

Reply:
Accepted. Thanks!

Reply:
Also, I think we should have `master` here.
You can find that the previous merged PR failed in the master branch :-(

Reply:
I mean these workflows should be run when they are PRs to master.

Reply:
Accepted. Will add `master` branch.